src/widget/debug.c

```diff
@@ -0,0 +1,136 @@
+#include "SDL.h"
+
```

Comment:
I think this should go to `platform` rather than `widget`.

Also, the external sdl include should go after the internal includes.